### DISCUSSION TITLE
用DAV同步网盘，内存不断增加，一直不释放。

### DISCUSSION THREAD

**Discussion by vation-wang** [AlistGo/alist#1033] [Q&A]
我用群晖918+，用docker 安装的alist盘，公共盘挂载完毕，用群晖 cloud sync 里的DVA同步公共盘，同步过程中从24m 内存 一直增加到2G 然后群晖死机，docker容器被迫关闭。
![截图20220503135347](https://user-images.githubusercontent.com/62927955/166409941-11b9c127-3a42-48aa-b191-fd94ff9a6fa4.png)

请管理告诉下如何解决内存不释放的问题？

**Comment by anwen-anyi** [comment]
用的什么版本? 挂载了迅雷云盘?
